examples/basic/client/src/main.rs

```diff
@@ -6,56 +6,73 @@ use com::{
 use interface::{Food, IAnimal, ICat, IDomesticAnimal, IExample, CLSID_CAT_CLASS};
 
 fn main() {
+    // Initialize the COM apartment
     init_apartment(ApartmentType::SingleThreaded)
         .unwrap_or_else(|hr| panic!("Failed to initialize COM Library{:x}", hr));
+    println!("Initialized apartment");
 
+    // Get a `BritishShortHairCat` class factory
     let factory = get_class_object::<IClassFactory>(&CLSID_CAT_CLASS)
```

Comment:
Is there also an equivalent of `CoCreateInstance(Ex)`?

Reply:
This is a call to CoGetClassObject. We use `create_instance` below which directly maps to `CoCreateInstance`.